core/src/main/java/org/eclipse/hono/util/MessageHelper.java

```diff
@@ -598,4 +640,133 @@ public static Boolean isDeviceCurrentlyConnected(final Message msg) {
         }).orElse(Boolean.FALSE);
     }
 
+    /**
+     * Set the payload of the message.
```

Comment:
IMHO this should also point out that the given payload will be added to the message in a *Data* section